pkg/kubectl/cmd/apply_test.go

```diff
@@ -279,6 +281,64 @@ func TestApplyObject(t *testing.T) {
 	}
 }
 
+func TestApplyObjectOutput(t *testing.T) {
+	initTestErrorHandler(t)
+	nameRC, currentRC := readAndAnnotateReplicationController(t, filenameRC)
+	pathRC := "/namespaces/test/replicationcontrollers/" + nameRC
+
+	// Add some extra data to the post-patch object
+	postPatchObj := &unstructured.Unstructured{}
+	if err := json.Unmarshal(currentRC, &postPatchObj.Object); err != nil {
+		t.Fatal(err)
+	}
+	postPatchLabels := postPatchObj.GetLabels()
+	if postPatchLabels == nil {
+		postPatchLabels = map[string]string{}
+	}
+	postPatchLabels["post-patch"] = "value"
```

Comment:
we already have tests that verify the value is updated correctly on the server. the referenced issue was that the value printed by `kubectl apply` used the original object fetched from the server, not the patched object. this testcase is sufficient to ensure the object returned by the server from the patch operation is the one output